chanzuckerberg/czgenepi
# [ch150281] filter panel toggle

### Summary
- **What:** 
  - Add a toggle button for the filter panel
  - Create an `enum` type def for data table view
  - Genericize styling for the IconButton grey background circle
- **Why:** 
  - So users can show/hide the panel
  - In case we ever need to change the view name, we only need to update one location
  - Reusability = DRY
- **Ticket:** [[ch150281]](https://app.clubhouse.io/genepi/story/150281)

### Testing
1. Click the button
1. Ensure filter panel disappears
2. Click again
3. Ensure panel is shown

### Demos
![after](https://user-images.githubusercontent.com/7562933/129120098-8010d033-39ce-4034-aa7f-8e91211209ef.gif)

### Checklist
